calcom/cal.com
# Change location of booking

## What does this PR do?

Allows users to change the location of a booking. Email is sent to the organizer and attendees to notify them about the updated location. 

Fixes #2393 

And fixes the following current issues:
- Dropdown doesn't go outside of dialog
- Input field loses focus after invalid input 
- Address of in person meeting not shown in event-type settings
- Dialog does not close after rejecting a booking

**Demo**
[Loom Video](https://www.loom.com/share/6740efe512ce47d5b95253649786252e)

## Type of change




## How should this be tested?

1. Go to upcoming bookings
2. Click ‘Edit booking’ and then ‘Change location’
3. Here you should see the current location of the booking
4. Choose a new location and click ‘update’
5. Go back to ‘Change location’, now current location should be updated 
6. Notification emails about the update are sent to the organizer and all attendees

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
